test/expected/create_chunks.out

```diff
@@ -64,7 +56,9 @@ ORDER BY c.id, d.id;
  _hyper_1_3_chunk |            2 |        3 |  1073741823 | 2147483647
 (6 rows)
 
+\c single super_user
```

Comment:
Reminds me that we should probably create a function that allows a user to change `num_slices`, including doing proper permissions checks for the hypertable.